test_pad_frame/test_pad_frame_genesis2.py

```diff
@@ -0,0 +1,29 @@
+from pad_frame import pad_frame_genesis2
+import glob
+import os
+
+
+def teardown_function():
+    for item in glob.glob('genesis_*'):
+        os.system(f"rm -r {item}")
+
+
+def test_main(capsys):
+    argv = [
+        "--config_data_width", "32",
```

Comment:
any reason we are using this? these are all defaults right?

Reply:
Not really. I basically just copied the memory core wrapper test, which did the same thing. Maybe at some point you'd want to change the parameterization you're testing without changing the defaults in the wrapper.